packages/blocks/src/api/constants.js

```diff
@@ -99,6 +99,7 @@ export const __EXPERIMENTAL_STYLE_PROPERTY = {
 			paddingBottom: 'bottom',
 			paddingLeft: 'left',
 		},
+		useEngine: true,
 	},
```

Comment:
Ideally at the end of the style engine implementation, this "constant" might be removed entirely if we did our work properly :)